source/fe/fe_pyramid_p.cc

```diff
@@ -103,6 +103,47 @@ FE_PyramidPoly<dim, spacedim>::FE_PyramidPoly(
       this->unit_support_points.emplace_back(-1.0, +1.0, 0.0);
       this->unit_support_points.emplace_back(+1.0, +1.0, 0.0);
       this->unit_support_points.emplace_back(+0.0, +0.0, 1.0);
+
+      this->unit_face_support_points.resize(5);
+
+      for (unsigned int f = 0; f < 5; ++f)
```

Comment:
Can we replace the hard-coded 5 in these two places by `ReferenceCells::Pyramid::n_faces()`? I think that would make the intent clearer.